go.mod

```diff
@@ -0,0 +1,25 @@
+module teleport-exec
+
+go 1.17
+
+require (
+	github.com/smartystreets/goconvey v1.7.2
```

Comment:
This is a bit of a heavy dependency for a small number of unit tests, and doesn't play well with standard Go tests. Fine for this project, but if it were me I would choose something much simpler.

Reply:
I prefer BDD-style test syntax of goconvey to the TestUnit-style of vanilla go tests (same in Ruby or Scala or any other language I've used), so decided to use it here to make it easier to develop the code. And since the dependency is not packaged anywhere, I do not see much harm in picking the tool that improves my experience as a developer.